platform/admin/ui/src/main/webapp/js/views/Alerts.view.js

```diff
@@ -60,7 +69,7 @@ define([
                         json.level = 'danger';
                         break;
                     default:
-                        json.level = 'info';
+                        json.level = 'danger';
```

Comment:
❓ are you sure changing this default doesn't impact any other alerts that were relying on the default being `info` level?